packages/cli/src/validators/getUnappliedSchemaOverridesTableNames.ts

```diff
@@ -0,0 +1,33 @@
+import { SchemaDefOverrides } from '../types/CliConfig';
+
+export function getUnappliedSchemaOverridesTableNames(
+    tables: string[],
+    defaultSchema: string,
+    schemaDefOverrides?: SchemaDefOverrides,
+) {
+    const unappliedSchemaOverrideTableKeys: string[] = [];
+
+    if (isObject(schemaDefOverrides)) {
+        const tablesToOverride = Object.keys(schemaDefOverrides);
+
+        for (const tableKey of tablesToOverride) {
+            const [schemaName, tableName] = tableKey.split('.');
+
+            if (schemaName === defaultSchema) {
+                if (!tables.includes(tableName)) {
+                    unappliedSchemaOverrideTableKeys.push(`- ${tableKey}`);
+                }
+            } else {
+                if (!tables.includes(tableKey)) {
+                    unappliedSchemaOverrideTableKeys.push(`- ${tableKey}`);
+                }
+            }
+        }
+    }
+
+    return unappliedSchemaOverrideTableKeys;
+}
+
+export function isObject(value: unknown): value is object {
```

Comment:
Are we using this anywhere else? if not, we should remove it as it's not really needed in this function.